docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
-
-**kubelet**
-
-To build the *kubelet*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kubelet*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet`
-   2. Windows: `go build cmd/kubelet/kubelet.go`
-
-**kube-proxy**
-
-To build *kube-proxy*, run:
-
-1. `cd $GOPATH/src/k8s.io/kubernetes`
-2. Build *kube-proxy*
-   1. Linux/MacOS: `KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy`
-   2. Windows: `go build cmd/kube-proxy/proxy.go`
+2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+3. [Optional] CNI network plugin installed.
 
-### Route Setup
-The below example setup assumes one Linux and two Windows Server 2016 nodes and a cluster CIDR 192.168.0.0/16
-
-| Hostname | Routable IP address | Pod CIDR |
-| --- | --- | --- |
-| Lin01 | `<IP of Lin01 host>` | 192.168.0.0/24 |
-| Win01 | `<IP of Win01 host>` | 192.168.1.0/24 |
-| Win02 | `<IP of Win02 host>` | 192.168.2.0/24 |
-
-**Lin01**
-
-```
-ip route add 192.168.1.0/24 via <IP of Win01 host>
-ip route add 192.168.2.0/24 via <IP of Win02 host>
-```
+**Windows Host Setup**
 
-**Win01**
+1. Windows Server container host running Windows Server version 1709 and Docker v17.06 or later. Follow the setup instructions outlined by this help topic: https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server.
+2. Build or download kubelet.exe, kube-proxy.exe, and kubectl.exe using instructions found [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+3. Copy Node spec file (config) from Linux master node with X.509 keys
+4. Create HNS Network
+5. Ensure correct CNI network config
+5. Start kubelet.exe using this script [start-kubelet.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/start-kubelet.ps1)
+6. Start kube-proxy using this script[start-kubeproxy.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/start-kubeproxy.ps1)
+7. [Optional] Add static routes on Windows host
 
-```
-docker network create -d transparent --gateway 192.168.1.1 --subnet 192.168.1.0/24 <network name>
-# A bridge is created with Adapter name "vEthernet (HNSTransparent)". Set its IP address to transparent network gateway
-netsh interface ipv4 set address "vEthernet (HNSTransparent)" addr=192.168.1.1
-route add 192.168.0.0 mask 255.255.255.0 192.168.0.1 if <Interface Id of the Routable Ethernet Adapter> -p
-route add 192.168.2.0 mask 255.255.255.0 192.168.2.1 if <Interface Id of the Routable Ethernet Adapter> -p
-```
-
-**Win02**
+**Windows CNI Config Example**
+Today, Windows CNI plugin is based on wincni.exe code with the following example, configuration file.
 
+Note: this file assumes that a user previous created 'l2bridge' host networks on each Windows node using <Verb>-HNSNetwork cmdlets as shown in the start-kublet.ps1 and start-kubeproxy.ps1 scripts linked above
```

Comment:
nit: minor grammar issues, also `<Verb>` is getting swallowed by the viewer (as HTML, I think?) so it just says "using -HNSNetwork." Escape it or wrap it in back-ticks. I'd wrap the script names in back-ticks too, for aesthetics :)

Reply:
ensure `start-kublet.ps1` is changed to `start-kubelet.ps1` , since the file was originally `start-kublet.ps1`

Reply:
Fixed these in https://github.com/JMesser81/kubernetes.github.io/pull/1